cross/ffmpeg-2.2.1/Makefile

```diff
@@ -0,0 +1,36 @@
+PKG_NAME = ffmpeg
+PKG_VERS = 2.2.1
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://www.ffmpeg.org/releases/
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/x264 cross/lame
+
+HOMEPAGE = http://www.ffmpeg.org/
+COMMENT  = FFmpeg is a complete, cross-platform solution to record, convert and stream audio and video. It includes libavcodec - the leading audio/video codec library
+LICENSE  =
+
+REQUIRED_DSM = 5.1
+
+# Define SYNCTHING_ARCH as per SyncThing standards
+ifeq ($(findstring $(ARCH),88f6281),$(ARCH))
+MYARCH = armv5
```

Comment:
yup if i remember correctly i've got an error like `armada is not a known arch` or something like that, will putt everything in `FFMPEG_ARCH=arm`